tests/spec/localhost/nvidia_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+describe command('nvidia-docker run --rm nvidia/cuda nvidia-smi') do
```

Comment:
using `nvidia-docker run --rm nvidia/cuda nvidia-smi -L` should reduce the length of the output if no further checks are added.